### DISCUSSION TITLE
Suggestions de réservations sur la page "Conflit entre réservation"

### DISCUSSION THREAD

**Discussion by Aurely9n** [JeromeDevome/GRR#310] [Idées]
Bonjour,

Lorsqu'on fait des réservations en masse, il peut y avoir des conflits.
Celles-ci sont bien affichées lorsque ça arrive mais dans ce cas là il n'y a que la possibilité de supprimer les anciennes réservations sans quoi, les nouvelles réservations ne sont pas enregistrées.
Y aurait-il un moyen d'enregistrer la nouvelle périodicité même lorsqu'il y a des conflits et de pouvoir gérer les conflits ensuite ? Via une page permettant de voir directement tous les conflits et des suggestions de modifications des réservations en questions ?
Il me semble qu'il y avait ce genre de fonctionnalité sur une version antérieure de GRR mais je n'en suis pas sur...

Le problème étant ici que c'est pénalisant de devoir supprimer/modifier les réservations en conflit avant de pouvoir enregistrer la nouvelle réservation périodique.

Cordialement

**Comment by ynaessens** [comment]
Bonjour,
la version 3 propose effectivement le choix entre supprimer l'ancienne ou sauter la nouvelle réservation.
À porter en version 4 si vous y tenez.
Cordialement,
YN

**Reply by Aurely9n** [reply]
Merci pour votre retour.

Si cela ne tient qu'à moi alors : OUI, j'y tiens ! 😄 
C'est plus pratique pour nous de valider l'enregistrement de la réservation périodique en sautant les itérations conflictuelles puis de géré manuellement ces dernières que d'être obligé de stopper la réservation, supprimer les conflits (qui peuvent parfois ne pas être supprimables) et revenir faire la réservation.

Cordialement,
Aurélien

**Comment by ynaessens** [comment]
Je n'en ai pas le temps actuellement, donc votre contribution est bienvenue si vous le voulez ou pouvez.
Cordialement,
YN
